packages/app/mirage/config.js

```diff
@@ -33,13 +33,15 @@ export default function() {
   this.passthrough('/write-coverage');
 
   // Mock bard facts + metadata
-  this.urlPrefix = `${config.navi.dataSources[0].uri}/v1`;
-  const metricBuilder = (metric, row, dimensionKey) => {
-    faker.seed(hashCode(`${row.dateTime}_${dimensionKey}_${metric}`));
-    return Number(faker.finance.amount());
-  };
-  BardLite.call(this, metricBuilder);
-  BardMeta.call(this);
+  config.navi.dataSources.forEach(({ uri }) => {
```

Comment:
We could probably add an if to do this only `type === 'bard'`

Reply:
Agreed, we only have bard today in app. I'll make an issue.

Reply:
https://github.com/yahoo/navi/issues/1111